DotNet-Platform.md

```diff
@@ -0,0 +1,40 @@
+![Logo][logo]
+
+# The .NET Platform 
+
+**.NET platform** is big and comprehensive collection of frameworks and libraries manager by [Microsoft](https://www.microsoft.com/net). In the beginning it could be quite daunting to get familiar with .Net ecosystem.
+
+Our aim is to get you familiar with the .Net platform, so that you are ready to learn the [c# programing language](C-Sharp) along with development environment, project structures, techniques and best practices you will be performing as a professional developer.
+
+Let's get some of the jargon's out of the way, don't worry, you don't need to know in-depth about these at this point.
```

Comment:
> Let's get some of the jargon's out of the way, don't worry, you don't need to know in-depth about these at this point.

That's not how commas are used in a sentence.

Make at least the first comma to be a period and start a new sentence.